src/containers/Settings/index.js

```diff
@@ -50,53 +48,49 @@ class Settings extends Component {
       />
       <CardContent>
         <List dense disablePadding>
-            <ListItem>
+          <ReactCopyToClipBoard text={didId}>
+            <ListItem button>
                 <ListItemIcon><UserIcon/></ListItemIcon>
                 <ListItemText
                     primary='Distributed Identity (DID)'
-                    secondary={didId}
+                    secondary={`${didId} - Click to copy to clipboard`}
+                />
+            </ListItem>
+          </ReactCopyToClipBoard>
+          <ReactCopyToClipBoard text={didPrivateKey}>
+            <ListItem button>
+                <ListItemIcon><KeyIcon/></ListItemIcon>
+                <ListItemText
+                    primary='DID Private Key'
+                    secondary='Click to copy to clipboard'
                 />
             </ListItem>
-            <ListItem>
+          </ReactCopyToClipBoard>
+          <ReactCopyToClipBoard text={address}>
+            <ListItem button>
                 <ListItemIcon><WalletIcon/></ListItemIcon>
                 <ListItemText
                     primary='Bitcoin Wallet (testnet)'
```

Comment:
We should call this "Your bitcoin address", just to avoid being caught off guard with the incorrect nomenclature.

Reply:
Addressed